app/views/notifications/_notification.html.erb

```diff
@@ -1,4 +1,4 @@
-<div role="list" class="w-full max-w-xl mx-auto rounded-lg overflow-hidden border border-gray-200 divide-y divide-gray-200 mt-3">
+<div role="list" class="w-full max-w-xl mx-auto sm:rounded-lg overflow-hidden border border-gray-200 divide-y divide-gray-200 sm:mt-3">
```

Comment:
```suggestion
<div role="list" class="w-full max-w-xl mx-auto sm:rounded-lg overflow-hidden border border-gray-200 divide-y divide-gray-200 mt-3">
```

Reply:
@joemasilotti commited the changes but this will keep the gap between notifications on mobile display.